src/libPMacc/include/pluginSystem/PluginConnector.hpp

```diff
@@ -180,6 +180,19 @@ namespace PMacc
             }
         }
 
+        template<typename Plugin>
+        Plugin* getPluginFromType()
```

Comment:
this chance should not be part of this PR but proposed/reviewed and merged beforehand

ccing @psychocoderHPC